provisioning/service/src/Twin/TwinCollection.cs

```diff
@@ -286,33 +286,20 @@ private void TryClearMetadata(string propertyName)
         }
 
         /// <summary>
-        /// Clears metadata out of the twin collection.
+        /// Clears all metadata out of the twin collection as well as the base metadata object.
         /// </summary>
         /// <remarks>
-        /// This will only clear the metadata from the twin collection but will not change the base metadata object. This allows you to still use
-        /// methods such as <see cref="GetMetadata"/>. If you need to remove all metadata, please use <see cref="ClearAllMetadata"/>.
+        /// This will remove all metadata from the base metadata object as well as the metadata for the twin collection. 
+        /// This will also clear the underlying metadata object which will affect methods such as
+        /// <see cref="GetMetadata"/> and <see cref="GetLastUpdatedVersion"/>.
         /// </remarks>
         public void ClearMetadata()
         {
             TryClearMetadata(MetadataName);
             TryClearMetadata(LastUpdatedName);
             TryClearMetadata(LastUpdatedVersionName);
             TryClearMetadata(VersionName);
-        }
-
-        /// <summary>
-        /// Clears all metadata out of the twin collection as well as the base metadata object.
-        /// </summary>
-        /// <remarks>
-        /// This will remove all metadata from the base metadata object as well as the metadata for the twin collection. The difference from the
-        /// <see cref="ClearMetadata"/> method is this will also clear the underlying metadata object which will affect methods such as
-        /// <see cref="GetMetadata"/> and <see cref="GetLastUpdatedVersion"/>.
-        /// This method would be useful if you are performing any operations that require <see cref="TryGetMemberInternal(string, out object)"/>
-        /// to return a <see cref="JToken"/> regardless of the client you are using.
-        /// </remarks>
-        public void ClearAllMetadata()
```

Comment:
There are 2 copies of this file, one in hub service client and another in dps service client. Please make the change in both.